src/Schema/Directives/Args/RulesDirective.php

```diff
@@ -15,4 +20,29 @@ public function name()
     {
         return 'rules';
     }
+
+    /**
+     * Resolve the field directive.
+     *
+     * @param ArgumentValue $value
+     *
+     * @return ArgumentValue
+     */
+    public function handleArgument(ArgumentValue $value)
+    {
+        // Mutation arguments are handled in the RuleFactory. This check
+        // should be unnecessary when additional interfaces are created for
+        // more fine-grain control.
+        if ('Mutation' === $value->getField()->getNodeName()) {
```

Comment:
Why differentiate between Queries and Mutations here? They are actually handled the same in RuleFactory as of #166